ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -9,21 +9,21 @@ set -x
 # or retries_max is reached.
 # Takes 2 arguments - the command/function and the output file
 run_scorecard() {
-  local retries_max=3
-  local attempt_num=1
-  until [ -s "$2" ]
-
-  do
-    if (( attempt_num>retries_max )); then
```

Comment:
I liked what we had before where we'd print out that maximum retries were reached. Also, I liked that earlier when we'd retry, we'd also print out the attempt number. We can still achieve that with the current refactoring.